### ISSUE TITLE
Improve disallowed-element message readability

### ISSUE THREAD

**Issue by YusukeHirao** [markuplint/markuplint#3815]
## Summary

`disallowed-element` reports violations with the raw selector string embedded in the message:

```
The "meta[charset] ~ meta[charset]" element is disallowed
```

When the rule is used through preset virtual rules (e.g. `markuplint:html-standard`'s `no-duplicate-charset`, `no-duplicate-description`, `no-charset-http-equiv-coexist`, `no-small-in-heading`), users see CSS selector syntax in the diagnostic instead of a spec-style explanation. This is hard to read for end users and leaks an implementation detail of how the constraint is encoded.

## Where this surfaces

`@markuplint/rules` `disallowed-element` rule (`packages/@markuplint/rules/src/disallowed-element/index.ts`):

```ts
const message = t('{0} is disallowed', t('the "{0*}" {1}', query, 'element'));
```

`query` is a CSS selector string, so the rendered message reads "The 'meta[charset] ~ meta[charset]' element is disallowed."

## Proposal

Support an optional human-readable label when `disallowed-element` is configured, and surface it in messages instead of the raw selector. The preset virtual rules that wrap `disallowed-element` already have a meaningful name (e.g. `html-standard/no-duplicate-charset`); reuse that label, or accept an explicit `reason` / `message` option.

A possible option shape (no commitment yet):

```jsonc
{
  "disallowed-element": {
    "value": [
      {
        "selector": "meta[charset] ~ meta[charset]",
        "message": "More than one meta charset is disallowed"
      }
    ]
  }
}
```

When the rule fires through a preset's virtual rule, the engine could use the virtual rule name as a fallback message when no explicit `message` is supplied.

## Spec reference

- HTML LS - meta element: https://html.spec.whatwg.org/multipage/semantics.html#the-meta-element

## Background

Surfaced during the audit for #3634 (resolved in #3814). End users running with `markuplint:html-standard` see violations like `The "meta[charset] ~ meta[charset]" element is disallowed`, which reads awkwardly. The benchmark itself does not care about the wording, but end-user UX does.
